Formula/postgresql.rb

```diff
@@ -94,11 +94,13 @@ def install
   end
 
   def post_install
+    require "open3"
     (var/"log").mkpath
     (var/"postgres").mkpath
     unless File.exist? "#{var}/postgres/PG_VERSION"
       system "#{bin}/initdb", "#{var}/postgres"
     end
+    _stdout, _stderr, _status = Open3.capture3("createdb")
```

Comment:
There's a chance of that but it seems that it handles the most typical new installation case so I think it's better. The only other option is a test for the user database existing if you have any ideas.